src/configure_edison

```diff
@@ -192,8 +192,7 @@ def _getGadgetMode():
     modestr = ''
     modestr = subprocess.run('lsusb', shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     modestr = str(modestr.stderr, 'utf-8')
-    gadget = modestr.find("unable to initialize libusb")
-    if gadget == -1:
+    if modestr.find("unable to initialize libusb") >= 0:
```

Comment:
This is opposite. I used to detect if the usb was in gadget mode using lsusb, because it threw the error. So "unable to initialize libusb" means we are in gadget mode. But I see now (zeus) we don't get an error at all, just silcence.

We need something better.

Reply:
Identified an alternative which works on the web interface: `dmesg | grep -q "usb0: link becomes ready"`.